src/app/static/src/app/root.ts

```diff
@@ -164,7 +164,7 @@ export const emptyState = (): RootState => {
 const existingState = localStorageManager.getState();
 
 export const storeOptions: StoreOptions<RootState> = {
-    state: {...emptyState(), ...existingState && {language: existingState.language}},
```

Comment:
I have always wondered why language was specifically set here when it's already part of ...existingState objects.